Security page and smaller updates (#10492)
DataDog/documentation: content/en/synthetics/browser_tests/actions.md

At recording, this translates into the actual value of the variable being injected on your website's input (consequently allowing you to move on with the rest of your steps) and creates an associated `Type text` step featuring `{{ <YOUR_VARIABLE_NAME> }}`.
At test execution, `{{ <YOUR_VARIABLE_NAME> }}` is systematically replaced by your variable's associated value.

> **Note**: Some variables only get computed at runtime (e.g. variable from HTTP request, variable from JavaScript step). To record a step using one of these variables, record a step with the actual variable value, then replace the actual value with `{{ <YOUR_VARIABLE_NAME> }}` on your step definition before saving your test. 

Reviewer: ```suggestion
**Note**: Some variables only get computed at runtime (For example, variable from HTTP request or variable from JavaScript step). To record a step using one of these variables, record a step with the actual variable value, then replace the actual value with `{{ <YOUR_VARIABLE_NAME> }}` on your step definition before saving your test. 
```
